src/asciidoc/tutorials/getting-started/quickStart.adoc

```diff
@@ -9,7 +9,7 @@ Along the way, you'll learn some general web development topics along with the f
 NOTE: This tutorial provides a step-by-step introduction for developers who are new to Koji, with or without web development experience.
 
 ////
-If you prefer to jump straight into code, you can head to the <<start-guide-1#,advanced quick start guide>>.
+If you prefer to jump straight into code, go through <<quick-start#, the Koji quick-start tutorial>>.
```

Comment:
Actually, let's revert this change. If you want, you can update the link to the slug you're going to use for the Advanced quick start guide (your next project). You can uncomment this part of the note as part of that ticket.

Reply:
Since it's commented out, I restored the text but I prepended "adv-" to the stub (adv-quick-start) and that's what I'll use for the new page. Thanks!